.buildkite/filebeat/filebeat-pipeline.yml

```diff
@@ -168,6 +168,9 @@ steps:
         artifact_paths:
           - "filebeat/build/*.xml"
           - "filebeat/build/*.json"
+          - "filebeat/build/integration-tests/*"
+          - "filebeat/build/integration-tests/Test*/*"
+          - "filebeat/build/integration-tests/Test*/data/**/*"
```

Comment:
Out of curiosity, what's the reason for not using `/**/*` instead?

Reply:
Just following a pattern that was already there and to be honest, I don't know if `build/integration-tests` or `build/integration-tests/Test*/data/` might have other folders we don't want

Reply:
See https://buildkite.com/docs/agent/v3/cli-artifact#uploading-artifacts-artifact-upload-examples:

```
Uploading all the files and folders inside the coverage directory:

buildkite-agent artifact upload "coverage/**/*"
```

IIUC, `filebeat/build/integration-tests/**/*` should archive all the files/folder/nestedfolder under `filebeat/build/integration-tests/`

Reply:
But gotcha, so if we don't need all the files/folders then, let's keep it simple